src/app/components/normal-component/chat/chat.component.ts

```diff
@@ -14,8 +15,8 @@ import { faReply, IconDefinition } from '@fortawesome/free-solid-svg-icons';
 export class ChatComponent implements OnInit, AfterViewChecked, OnDestroy {
     public static VERBOSE: boolean = false;
 
-    @Input() public chatId: string;
-    @Input() public turn: number;
+    @Input() public chatId!: string;
+    @Input() public turn!: number;
```

Comment:
what's that ?

Reply:
en préparation pour la suite de strictness (qui est pas pour tout de suite no worries!), c'est une bonne idée d'utiliser le postfixe `!` pour dire "ceci est une variable qui sera TOUJOURS initialisée et donc jamais nulle" (https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-7.html#definite-assignment-assertions)

Reply:
ah oui, non NonNullable pour les input donc, je vois !